pyriemann/utils/tangentspace.py

```diff
@@ -120,8 +117,17 @@ def exp_map_logeuclid(X, Cref):
     Notes
     -----
     .. versionadded:: 0.4
+
+    References
+    ----------
+    .. [1] `A New Canonical Log-Euclidean Kernel for Symmetric Positive
+        Definite Matrices for EEG Analysis
+        <https://ieeexplore.ieee.org/document/10735221>`_
+        G. Wagner vom Berg, V. Röhr, D. Platt, B. Blankertz. IEEE TBME, 2024.
     """
-    return expm(X + logm(Cref))
+    d, V = np.linalg.eigh(Cref)
+    logfdd = _first_divided_difference(d, np.log, lambda x: 1 / x)
+    return expm(logm(Cref) + V @ (logfdd * (V.conj().T @ X @ V)) @ V.conj().T)
```

Comment:
I think it's interesting to add a function to calculate the directional derivative of the matrix logarithm:
```
def ddlogm(X, Cref):
    d, V = np.linalg.eigh(Cref)
    logfdd = _first_divided_difference(d, np.log, lambda x: 1 / x)
    return V @ (logfdd * (V.conj().T @ X @ V)) @ V.conj().T
```
Same remark for the directional derivative of the matrix exponential.